scripts/just-task.js

```diff
@@ -56,26 +56,28 @@ task('ts', parallel('ts:commonjs', 'ts:esm', condition('ts:amd', () => argv().pr
 
 task(
   'build',
-  series(
-    'clean',
-    'copy',
-    'sass',
-    parallel(
-      condition('tslint', () => !argv().min && !argv().prdeploy),
-      condition('jest', () => !argv().min && !argv().prdeploy),
-      series(
-        argv().commonjs ? 'ts:commonjs-only' : 'ts',
-        condition('lint-imports', () => !argv().min && !argv().prdeploy),
-        parallel(condition('webpack', () => !argv().min), condition('verify-api-extractor', () => !argv().min && !argv().prdeploy))
+  condition(
+    series(
+      'clean',
+      'copy',
+      'sass',
+      parallel(
+        condition('tslint', () => !argv().min && !argv().prdeploy),
+        condition('jest', () => !argv().min && !argv().prdeploy),
+        series(
+          argv().commonjs ? 'ts:commonjs-only' : 'ts',
+          condition('lint-imports', () => !argv().min && !argv().prdeploy),
+          parallel(condition('webpack', () => !argv().min), condition('verify-api-extractor', () => !argv().min && !argv().prdeploy))
+        )
       )
-    )
+    ),
+    ignoreNpmInstallBuildInCI
```

Comment:
What does this arg do?

Reply:
save it from building initially in npm install step in CI